crates/e2e/src/drink_client.rs

```diff
@@ -199,11 +209,17 @@ where
 impl<
         AccountId: Clone + Send + Sync + From<[u8; 32]> + AsRef<[u8; 32]>,
         Hash: Copy + From<[u8; 32]>,
-        Runtime: RuntimeT,
-        E: Environment<AccountId = AccountId, Balance = u128, Hash = Hash> + 'static,
+        Runtime: RuntimeT + pallet_balances::Config + pallet_contracts::Config,
+        E: Environment<
+                AccountId = AccountId,
+                Balance = ContractsBalanceOf<Runtime>,
+                Hash = Hash,
+            > + Send
```

Comment:
Why do you need `Send` here?

Reply:
We don't, I mistakenly added it. Removed!